inst/NEWS

```diff
@@ -3,4 +3,5 @@ Changes for v0.99
 
 - compliance for bioconductor
 - qza_to_phyloseq is less strict about objects included in phyloseq object
-- added vignette and help
\ No newline at end of file
+- added vignette and help
+- the function qza_to_tse has been added, facilitating operability between qiime2R and microbiome packages
```

Comment:
"microbiome packages" might be misleading as it refers to a phyloseq-based "microbiome" package

Reply:
-> write instead: "facilitating operability between qiime2R and R/Bioc packages suporting TreeSummarizedExperiment"

Reply:
done